src/middlewared/middlewared/plugins/truenas_connect/finalize_registration.py

```diff
@@ -88,6 +88,17 @@ async def registration(self, job):
                             'registration_details': decoded_token,
                         }
                     )
+
+                    # Register system configuration (including websocket port) with TNC
+                    logger.debug('Registering system configuration with TNC')
```

Comment:
Needs to be when ui_https_port changes, not post registration, since that configuration covered with the POST on registration.